### ISSUE TITLE
Cannot use `\ref` in appendix proofs with the `commandchars` argument of `\fvset` with IEEEtran

### ISSUE THREAD

**Issue by a3nm** [PierreSenellart/apxproof#40]
Hi,

I cannot compile the following document:

```
\documentclass{IEEEtran}
\usepackage{apxproof}
\fvset{commandchars=\\\{\}}
\newtheoremrep{theorem}{Theorem}[section]

\begin{document}

\section{Test}
\begin{theoremrep}
  X
\end{theoremrep}
 
\begin{proof}
  \ref{a}
\end{proof}
  
\end{document}
```

It produces the following error:

```
Runaway text?
  \protect \gdef \@refundefined {\protect \begingroup \def \MessageBreak \ETC.
! Forbidden control sequence found while scanning text of \write.
<inserted text> 
                }
l.14   \ref{a}
```

However, either commenting away the `\fvset` command or removing the `\ref` in the proof (and putting just text instead) compiles fine.

Thanks if you can help!

**Comment by PierreSenellart** [comment]
Fixed in c9a79573a16fa9de3685ca180e5e4f7be0f34655; can you test?

**Comment by a3nm** [comment]
Yes, this works, thanks a lot!
